ui/imports/shared/status/StatusEmojiPopup.qml

```diff
@@ -195,6 +195,7 @@ Popup {
     }
 
     onOpened: {
+        localAccountSensitiveSettings.recentEmojis =[];
```

Comment:
@alexandraB99 this means on every `StatusEmojiPopup` opening stored recent emojis will be deleted, is that an intention in this case? If we need to keep previously stored emojis this should be fixed in a proper way also.

Reply:
removed

Reply:
Turns out we need a way to initialize this with an empty array by default though, because without it, it's gonna be an empty string, which breaks the emoji selection code.